Blog/include/functions.php

```diff
@@ -54,6 +54,27 @@ function printHeader(){
 		echo "<html>";
 };
 
+function getAllTags(){
+	$result = dbQuery("
+		SELECT *
+		FROM Tags
+	")->fetchAll();
+
+	return $result;
+}
+
+
+function getTagBlogs($tagID){
+	$result = dbQuery("
+		SELECT *
+		FROM PostTag
```

Comment:
It looks to me like this is only retrieving the PostTag table which I assume only has a `postId` and `tagId` field. It doesn't contain info on the actual tags or the actual posts, right?

If you join this query with the post table, you'd be able to see all the info about the posts.

Side note: for some reason you're not supposed to use capital letters in table names. I'm not sure why, but it's probably a good idea to rename your tables so they're all lowercase (for mysql tables I use the `post_tag` format)